KUBERNETES.md

```diff
@@ -236,7 +250,12 @@ helm install \
   --set database=postgresql \
   --set postgresql.enabled=true \
   --set postgresql.replication.enabled=true \
-  --set postgresql.replication.slaveReplicas=3
+  --set postgresql.replication.slaveReplicas=3 \
+  --set createSecret=true \
+  --set createRabbitMqSecret=true \
+  --set createRedisSecret=true \
+  --set createMysqlSecret=true \
```

Comment:
We don't need the mysql secret of we use postgresql. Same for Redis.

Reply:
yes I know but it seemed more simple to just give the command that always works.

Reply:
I have added the info in the doc. hopefully that will make travis happy too, not sure what happened with travis